sw/device/lib/testing/test_rom/test_rom.c

```diff
@@ -55,6 +56,9 @@ static dif_rv_core_ibex_t ibex;
 // rom tests. By default, it simply jumps into the OTTF's flash.
 OT_WEAK
 bool rom_test_main(void) {
+  // Initial sec_mmio, required by bootstrap and its dependencies.
+  sec_mmio_init();
```

Comment:
Does this have an observable simulation time penalty? We may want to consider a lightweight init sequence exclusively for DV. @timothytrippel would know the answer here.

Reply:
Good point. If it ends up being an issue, we can probably just init the counters and skip table initialization (will be a separate test-only library). @timothytrippel plmk if you notice any significant overhead after this change.

Reply:
Yeah dropping the table initialization would help, but we can keep an eye on this and optimize if needed.